osmaxx/core/templates/base.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+{% load staticfiles %}
+
+<html>
+<head lang="en">
+    <meta charset="UTF-8">
+    <title>Osmaxx</title>
+    <link rel="icon" type="image/png" href='{% static "osmaxx/images/favicon.png" %}' />
```

Comment:
Would title and favicon belong into `osmaxx/base.html` instead of here in `base.html`? Everything else here looks generic enough to be independent of the OSMaxx project.

If not, what is the rule (of thumb) for what belong here, and what into `osmaxx/base.html`?